Sources/Commands/SwiftBuildTool.swift

```diff
@@ -173,13 +173,13 @@ public struct SwiftBuildTool: SwiftTool {
         print("USAGE: swift build [mode] [options]")
         print("")
         print("MODES:")
-        print("  -c, --configuration <value>   Build with configuration (debug|release)")
-        print("  --clean <mode>                Delete artifacts (build|dist)")
+        print("  -c, --configuration <value>   Build with configuration (debug|release) [default: debug]")
+        print("  --clean <mode>                Delete artifacts (build|dist) [default: build]")
```

Comment:
That's correct, AFAIU what `default` means for Modes and Options is slightly different. If you pass a flag for a different *mode* (`--configuration`, `--clean`), it runs with the default parameter specified in the brackets; instead of building (the default mode).

If you pass an *option* flag without specifying its value (`--build-path`, `--color`), that's a syntax error.

Is this accurate, @aciidb0mb3r?